packages/core/strapi/src/middlewares/security.ts

```diff
@@ -13,7 +13,8 @@ const defaults: Config = {
   contentSecurityPolicy: {
     useDefaults: true,
     directives: {
-      'connect-src': ["'self'", 'https:'],
+      'script-src': ["'self'", "'unsafe-inline'"],
+      'connect-src': ["'self'", 'ws:', 'https:'],
```

Comment:
@innerdvations I had to add these to get the Vite client to work correctly with it's websockets, can I add these dynamically in the develop command?

I'm under the assumption it's not great to add these as defaults and wondering what the better way would be 😄 

cc @alexandrebodin if you get here first.

Reply:
For reference vite also injects an inline script for the HMR stuff.

Reply:
We might be able to programaticaly add that config before the server starts 🤔 we can look at it together if you want